include/revng/Pypeline/Helpers/Python/Registry.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <vector>
+
+#include "nanobind/nanobind.h"
+
+namespace revng::pypeline::helpers::python {
+
+struct BaseClasses {
+  nanobind::object BaseContainer;
+  nanobind::object BaseAnalysis;
+  nanobind::object BasePipe;
+};
+
+class RegistryImpl {
+private:
+  using PythonModuleInitializer = void (*)(nanobind::module_ &, BaseClasses &);
+  std::vector<PythonModuleInitializer> ModuleInitializers;
+
+public:
+  RegistryImpl() = default;
+  ~RegistryImpl() = default;
```

Comment:
Drop

Reply:
Isn't this part of the rule of 0/3/5? Shouldn't they all be present?